#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("n_clusters", int_t, 0, "n_clusters", 10, 1, 300)

exit(gen.generate(PACKAGE, "jsk_perception", "KMeans"))
